packages/headless/src/entity/cart.ts

```diff
@@ -0,0 +1,14 @@
+import { ObjectId } from 'mongoose';
+
+export class Cart {
+  id: string;
+  user: string;
+  items: [Item];
+  createdOn: Date;
+  updatedOn: Date;
```

Comment:
we can use mongoose timestamp flag which will automatically add createdAt and updatedAt property and that fields will be maintained by mongoose